Sources/xpmcore/Utils/Glob.swift

```diff
@@ -0,0 +1,214 @@
+//
+//  Created by Eric Firestone on 3/22/16.
+//  Copyright © 2016 Square, Inc. All rights reserved.
+//  Released under the Apache v2 License.
+//
+//  Adapted from https://gist.github.com/efirestone/ce01ae109e08772647eb061b3bb387c3
+import Foundation
+
+public let GlobBehaviorBashV3 = Glob.Behavior(
```

Comment:
The glob implementation that we were using from PathKit doesn't expand the directories recursively for the `**` path component.